taxcalc/params.json

```diff
@@ -920,9 +920,9 @@
     },
     "_NIIT_trt":{
         "long_name": "Net Investment Income Tax rate (UIMC)",
-	    "description": "AKA Unearned Income Medicare Distribution Tax. Taxed on net investment income of individuals with MAGI higher than certain thresholds.",
-        "irs_ref": "Form , line , ",
-        "notes": "HC",
+	"description": "Also called Unearned Income Medicare Contribution Tax. You will be taxed at this rate if your MAGI is higher than NIIT threshold.",
```

Comment:
On Tue, 4 Aug 2015, Matt Jensen wrote:

> 
> In taxcalc/params.json:
> 
> > @@ -920,9 +920,9 @@
> >      },
> >      "_NIIT_trt":{
> >          "long_name": "Net Investment Income Tax rate (UIMC)",
> > -	    "description": "AKA Unearned Income Medicare Distribution Tax. Taxed 
> on net investment income of individuals with MAGI higher than certain thresh
> olds.",
> > -        "irs_ref": "Form , line , ",
> > -        "notes": "HC",
> > +	"description": "Also called Unearned Income Medicare Contribution Tax. Yo
> u will be taxed at this rate if your MAGI is higher than NIIT threshold.",
> 
> Is this right? ->"Also called Unearned Income Medicare Contribution. Your
> MAGI above the NIIT threshold will be taxed at this rate."

I would suggest:  "Also called the Unearned Income Medicare Contribution 
Tax. The lessor of net investment income and the amount by which MAGI 
exceeds a threshold is taxed at this rate."

Should we strive to eliminate the "you" and "your" from these 
descriptions? Shouldn't it be "taxpayer"?

dan

> 
> —
> Reply to this email directly or view it on
> GitHub.[AHvQVcD2zBhvtMm0xQOVF4CJJ0B7JJA8ks5okKHFgaJpZM4FhcK0.gif]
> 
> 
>